packages/salesforcedx-vscode-lwc/src/testSupport/workspace/getLwcTestRunnerExecutable.ts

```diff
@@ -17,13 +16,17 @@ import { workspaceService } from './workspaceService';
  * @param cwd path to the workspace folder
  * @returns path to LWC Test runner
  */
-export const getLwcTestRunnerExecutable = (cwd: string) => {
+export const getLwcTestRunnerExecutable = async (cwd: string) => {
   const workspaceType = workspaceService.getCurrentWorkspaceType();
   if (workspaceService.isSFDXWorkspace(workspaceType)) {
     const lwcTestRunnerExecutable = path.join(cwd, 'node_modules', '.bin', 'lwc-jest');
-    if (fs.existsSync(lwcTestRunnerExecutable)) {
-      return lwcTestRunnerExecutable;
-    } else {
+    try {
+      const uri = vscode.Uri.file(lwcTestRunnerExecutable);
+      const stat = await vscode.workspace.fs.stat(uri);
```

Comment:
starting to feel a bit repetitive...now that you've seen several of these PRs, how do you feel about having a module in vscode-utils that provides the vscode equivalent of `exists` ?  

It seems like most of the `stat` checks are not really interested in the stat, just the existence.

Reply:
Sure, good idea, I'll take care of this with [W-18379555](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00002DZbwDYAT/view) as it's the last one I'm working on that is about the `fs` change